Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/StateSynchronization/ICommandHandler.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Extensions.Experimental.Socketer;
+using System.IO;
+
+namespace Microsoft.MixedReality.Toolkit.Extensions.Experimental.SpectatorView
+{
+    public interface ICommandHandler
+    {
+        void OnConnected(SocketEndpoint endpoint);
+        void OnDisconnected(SocketEndpoint endpoint);
+        void HandleCommand(string command, SocketEndpoint endpoint, BinaryReader reader);
```

Comment:
>string command, SocketEndpoint endpoint [](start = 27, length = 39)

Nit: I would reorder these parameters so the SocketEndpoint is always the first parameter for these 3 functions